This content has moved to https://commcare-cloud.readthedocs.io/en/latest/changelog/0027-fix-python3-virtualenv.html
